frontend/src/components/auth/Login.tsx

```diff
@@ -1,10 +1,17 @@
+import { gql, useMutation } from "@apollo/client";
 import React, { useContext, useState } from "react";
+import {
+  GoogleLogin,
+  GoogleLoginResponse,
+  GoogleLoginResponseOffline,
+} from "react-google-login";
 import { Redirect } from "react-router-dom";
-import { gql, useMutation } from "@apollo/client";
 import authAPIClient from "../../APIClients/AuthAPIClient";
 import AuthContext, { AuthenticatedUser } from "../../contexts/AuthContext";
 import ResetPassword from "./ResetPassword";
 
+type GoogleSucces = GoogleLoginResponse | GoogleLoginResponseOffline;
```

Comment:
Question: Is GoogleResponse an appropriate name? based on l120, I don't get the impression that this necessarily types a success